canonical/canonical.com
# WD-15046 Dev Created /data/support/spark

## Done

- Created /data/support/spark.
- Moved spark page to spark bubble.
- This is a new page.
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1430-7099&node-type=canvas&m=dev)
- [Copydoc](https://docs.google.com/document/d/1EcgdnMdW5jfZLLhcisUNMMtsPT-4alaPm3zirtvXEMk/edit?tab=t.0)
- [Demo](https://canonical-com-1403.demos.haus/data/spark/support)

## Issue / Card

Fixes # [WD-15046](https://warthogs.atlassian.net/browse/WD-15046)

## Screenshots

[if relevant, include a screenshot]


[WD-15046]: https://warthogs.atlassian.net/browse/WD-15046?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
